content/blog/2019-02-18-stats19.md

```diff
@@ -0,0 +1,1154 @@
+
+</th>
+
+</tr>
+
+</thead>
+
+<tbody>
+
+<tr>
+
+<td style="text-align:left;">
+
+2017010001708
+
+</td>
+
+<td style="text-align:right;">
+
+\-0.080107
+
+</td>
+
+<td style="text-align:right;">
+
+51.65006
+
+</td>
+
+<td style="text-align:left;">
+
+Fatal
+
+</td>
+
+<td style="text-align:left;">
+
+2017-08-05
+
+</td>
+
+</tr>
+
+<tr>
+
+<td style="text-align:left;">
+
+2017010009342
+
+</td>
+
+<td style="text-align:right;">
+
+\-0.173845
+
+</td>
+
+<td style="text-align:right;">
+
+51.52242
+
+</td>
+
+<td style="text-align:left;">
+
+Slight
+
+</td>
+
+<td style="text-align:left;">
+
+2017-01-01
+
+</td>
+
+</tr>
+
+<tr>
+
+<td style="text-align:left;">
+
+2017010009344
+
+</td>
+
+<td style="text-align:right;">
+
+\-0.052969
+
+</td>
+
+<td style="text-align:right;">
+
+51.51410
+
+</td>
+
+<td style="text-align:left;">
+
+Slight
+
+</td>
+
+<td style="text-align:left;">
+
+2017-01-01
+
+</td>
+
+</tr>
+
+<tr>
+
+<td style="text-align:left;">
+
+2017010009348
+
+</td>
+
+<td style="text-align:right;">
+
+\-0.060658
+
+</td>
+
+<td style="text-align:right;">
+
+51.62483
+
+</td>
+
+<td style="text-align:left;">
+
+Slight
+
+</td>
+
+<td style="text-align:left;">
+
+2017-01-01
+
+</td>
+
+</tr>
+
+<tr>
+
+<td style="text-align:left;">
+
+2017010009350
+
+</td>
+
+<td style="text-align:right;">
+
+\-0.072372
+
+</td>
+
+<td style="text-align:right;">
+
+51.57341
+
+</td>
+
+<td style="text-align:left;">
+
+Serious
+
+</td>
+
+<td style="text-align:left;">
+
+2017-01-01
+
+</td>
+
+</tr>
+
+<tr>
+
+<td style="text-align:left;">
+
+2017010009351
+
+</td>
+
+<td style="text-align:right;">
+
+\-0.353876
+
+</td>
+
+<td style="text-align:right;">
+
+51.43876
+
+</td>
+
+<td style="text-align:left;">
+
+Slight
+
+</td>
+
+<td style="text-align:left;">
+
+2017-01-01
+
+</td>
+
+</tr>
+
+</tbody>
+
+</table>
+
+The package contains the names of all “zip” files released by the DfT
+and hosted on Amazon servers to download. These file names have been
+included in the package and can be found under `file_names` variable
+name. for example:
+
+``` r
+stats19::file_names$dftRoadSafetyData_Vehicles_2017.zip
+```
+
+    ## [1] "dftRoadSafetyData_Vehicles_2017.zip"
+
+You can also get the raw data (if you really want\!) to see how much
+more useful the data is after it has been cleaned and labelled by the
+`stats19` package, compared with the data provided by
+government:
+
+``` r
+crashes_2017_raw = get_stats19(year = 2017, type = "Accidents", ask = FALSE, format = FALSE)
+```
+
+    ## Files identified: dftRoadSafetyData_Accidents_2017.zip
+
+    ##    http://data.dft.gov.uk.s3.amazonaws.com/road-accidents-safety-data/dftRoadSafetyData_Accidents_2017.zip
+
+    ## Data already exists in data_dir, not downloading
+
+    ## Data saved at /tmp/RtmpYh76XA/dftRoadSafetyData_Accidents_2017/Acc.csv
+
+    ## Reading in:
+
+    ## /tmp/RtmpYh76XA/dftRoadSafetyData_Accidents_2017/Acc.csv
+
+``` r
+kableExtra::kable(head(crashes_2017_raw[, c(1, 4, 5, 7, 10)]))
```

Comment:
I'm not sure how valuable it is to show this second table that is nearly identical to the earlier one but with different accident\_severity and date formats

It's more important in the post to have the story in mind that you want to tell with the example you have chosen

Reply:
OK. Maybe we could just show the first 6 values of `accident_severity` (or another variable) in the unformated form to show how hard it is to use the raw data. Sound reasonable? Agree the tables do duplicate some info.

Reply:
I had that feeling myself, see what I can do.

Reply:
Right, I think we might have a slight issue. I will see if I can override the CSS. 

The code highlighting css (I should know the name but I don't which looks great) uppercases and escapes table headers which in turn undoes what `stats19` format does on the `df` names. 

will let you know if this causes an issue.
link
view-source:https://d33wubrfki0l68.cloudfront.net/css/35e331193c647bac6aee0f2bb27072565a8474e3/css/style-new.css
image
![image](https://user-images.githubusercontent.com/408568/53087900-28a9ce80-3500-11e9-92e5-3dfc551b7509.png)

Reply:
OK, minor css changes and I think it should be fine.